merlin/celery.py

```diff
@@ -131,10 +137,13 @@ def setup(**kwargs):
     """
     if "CELERY_AFFINITY" in os.environ and int(os.environ["CELERY_AFFINITY"]) > 1:
         # Number of cpus between workers.
-        cpu_skip = int(os.environ["CELERY_AFFINITY"])
-        npu = psutil.cpu_count()
-        p = psutil.Process()
-        current = billiard.current_process()
-        prefork_id = current._identity[0] - 1  # range 0:nworkers-1
-        cpu_slot = (prefork_id * cpu_skip) % npu
-        p.cpu_affinity(list(range(cpu_slot, cpu_slot + cpu_skip)))
+        cpu_skip: int = int(os.environ["CELERY_AFFINITY"])
+        npu: int = psutil.cpu_count()
+        process: psutil.Process = psutil.Process()
+        # pylint is upset that typing accesses a protected class, ignoring W0212
+        # pylint is upset that billiard doesn't have a current_process() method - it does
+        # pylint: disable=W0212, E1101
```

Comment:
Do we want to re -enable these after? Same for the disable on 130 (W0613)

Reply:
Good catch, patched to disable-next.